.github/workflows/ci.yml

```diff
@@ -1,10 +1,33 @@
 ---
-name: Pulp CI
-on:
-  push:
-    branches:
-      - '*'
+name: Pulp PR
+on: pull_request
```

Comment:
Is this consistent with the plugin-template?

It's not consistent with the filename.

Reply:
Maybe I'm out of date but I thought the CI for the installer was totally independent from the plugin-template?

I can change the name to "Pulp CI" if that's what you mean

Reply:
I meant having only pull_request in ci.yml.

I see that it is more consistent with the plugin-template.

Reply:
There was some discussion about doing this (disabling CI on branch pushes) as well.  No quorum on that decision but we can certainly discuss doing that.